tests/ext/sandbox/exception_is_defined.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Exceptions in the tracing closure callback are always defined
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 50500) die('skip PHP 5.4 not supported'); ?>
+--FILE--
+<?php
+
+var_dump(error_get_last());
+dd_trace_function('array_sum', function ($span, $args, $retval, $ex) {
+	return \is_null($ex);
```

Comment:
Since we'll have a patch that prevents errors in tracing closures from making their way into `error_get_last()` from outside the closure, we could `var_dump($ex)` here and then check `error_get_last()` directly inside the closure. Something like:

```php
var_dump($e);
var_dump(error_get_last());
```

That way if the patch for `error_get_last()` were to land before this one, this test would still reliably target this issue:

```
NULL
array(4) {
  ["type"]=>
  int(8)
  ["message"]=>
  string(22) "Undefined variable: ex"
  ["file"]=>
  string(%d) "%s"
  ["line"]=>
  int(%d)
}
```